include/region_map.h

```diff
@@ -25,7 +26,7 @@ enum {
     NUM_MAPSEC_TYPES
 };
 
-struct RegionMap {
+typedef struct RegionMap {
```

Comment:
any reason you've changed this?

Reply:
good catch, this didn't need to be changed. tested and works without it!